content/announcements/2025-02-11-contribfest-kubecon-eu-2025.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: contribfest-kubecon-eu-2025
+title: Join Us for ContribFest at KubeCon EU 2025 in London!
+description: The cert-manager maintainers are hosting a ContribFest event on 2nd April, 2025, and you are invited!
+date: "2025-02-11T09:00:00Z"
+---
+
+
+
+Are you heading to KubeCon EU 2025 in London this April? If so, we have some exciting news: the cert-manager maintainers are hosting a [ContribFest event on Wednesday April 2, 2025 14:30 - 15:45 BST](https://kccnceu2025.sched.com/event/1tcxb/contribfest-dive-into-cert-manager-and-start-contributing) taking place on Level 3 in the ICC Capital Suite 1, and you are invited!
+
+As a newly graduated CNCF project, there's never been a better time to get involved with cert-manager and make an impact in the Kubernetes ecosystem.
+
+In this blog post, we will explain what ContribFest is and list first good issues to help you get started, whether you are joining us in London at KubeCon EU 2025 or participating from anywhere in the world.
+
+## What is ContribFest?
+
+ContribFest is an opportunity to join project maintainers and community contributors to explore good first issues, hunt bugs, discuss improvements, and pair program with maintainers to contribute directly to CNCF projects during a 90-minute session.
+
+At this year's cert-manager ContribFest, maintainers including CyberArk's Maël Valais and Richard Wall, and Zenior's Erik Godding Boye will be on hand to show users of all experience levels how to get involved with the cert-manager project.
+
+## Good First Issues
+
+TBD
```

Comment:
@SgtCoDFish I'll add this part later on.

Reply:
```suggestion
An updated list of open good first issues will be available in [this google sheet](https://docs.google.com/spreadsheets/d/1zThfUB22HHdHAiRvS3ctbj4Da7j30imnUleURjxTYE0/edit?usp=sharing).
```

I don't mind moving the sheet to a cert-manager owned place. Just in my Jetstack google account atm.